src/hypothesis/strategies.py

```diff
@@ -1036,10 +1036,11 @@ def data():
     """This isn't really a normal strategy, but instead gives you an object
     which can be used to draw data interactively from other strategies.
 
-    It can only be used within @given, not find. This is because the lifetime
-    of the object cannot outlast the test body.
+    It can only be used within @given, not find or @example. This is because
```

Comment:
The fact that it can't be used inside example is for a different reason than why it can't be used in find (it's just not been implemented, there's no fundamental limitation), so this doesn't fit with the next sentence.